app/main.py

```diff
@@ -1,8 +1,36 @@
+from typing import List, Dict
+
+
 class Person:
-    # write your code here
-    pass
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+    people = {}
```

Comment:
The `people` dictionary is defined as a class attribute. This means it is shared across all instances of the `Person` class. If you intend for each `Person` instance to have its own `people` dictionary, consider defining it as an instance attribute instead.